### PR TITLE
create: add --ruby option

### PR DESCRIPTION
**Pull request by dawidd6**
Make possible executing `brew create --ruby` to create basic template for building Ruby gems.

- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

### CODE REVIEW

File: Library/Homebrew/formula_creator.rb [hunk 1]
```diff
@@ -166,6 +168,12 @@ def install
             bin.env_script_all_files(libexec/"bin", :PERL5LIB => ENV["PERL5LIB"])
         <% elsif mode == :python %>
             virtualenv_install_with_resources
+        <% elsif mode == :ruby %>
+            ENV["GEM_HOME"] = libexec
+            system "gem", "build", "\#{name}.gemspec"
+            system "gem", "install", "\#{name}-\#{version}.gem"
```
Comment (MikeMcQuaid) [inline Library/Homebrew/formula_creator.rb]: I'm wondering if we should/could try to use `bundle install --standalone` here instead. That would avoid needing to `resource` all the dependent gems, I think?

Comment (dawidd6) [reply Library/Homebrew/formula_creator.rb]: All dependencies are being installed by `gem install` AFAIK. Couldn't find any example of a formula in core that uses `bundle install`.

Comment (MikeMcQuaid) [reply Library/Homebrew/formula_creator.rb]: Yeh, fair enough, that seems fine 👍

**Comment by dawidd6** [comment]
Any ideas why `brew tests` failed?

**Comment by MikeMcQuaid** [comment]
Looks like a flaky cask test. Could you create an issue and/or delete it?

**Comment by dawidd6** [comment]
Flaky test deleted.

**Comment by MikeMcQuaid** [comment]
Thanks again @dawidd6!

**Pull request merged**